src/ngraph_rewrite_pass.cc

```diff
@@ -209,6 +214,15 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
       return Status::OK();
     }
 
+#if defined(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
```

Comment:
Technically we dont need the ifdef perhaps? `ReplaceModifiers` should not have any effect in grappler or normal paths

Reply:
Yes, perhaps but I would suggest to keep the ifdef for clarity.